Add cangjie extension

jimmyclchu opened the pull request:
## Description

<!-- A summary of your change. If you add a new extension or command, explain what it does. -->

## Screencast

<!-- If you add a new extension or command, include a screencast (or screenshot for straightforward changes). A good screencast will make the review much faster - especially if your extension requires registration in other services.  -->

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

LitoMore commented:
Hi @jimmyclchu, thanks for contributing.

How about adding some actions, such as <kbd>Copy Characters</kbd>, <kbd>Copy Input Keys</kbd>, to the action list?

It might be helpful for users who want to share them.

pernielsentikaer commented:
Hi @jimmyclchu👋 

Thanks for your contribution 🔥

Could you look into the feedback 😊

jimmyclchu commented:
Hi @pernielsentikaer, typos are fixed and I added some more context in the readme. Apperciated for reviewing.

Thanks @LitoMore, good advice and I added these features.

jimmyclchu commented:
Of course. Thanks @LitoMore!
